content/en/docs/KRE/user/50_krt_v2/specs/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "KRT V2 YAML specs"
+description: >
+  YAML Definition of a KRT file.
+weight: 20
+---
+
+A KRT yaml file is a declarative file describing the content of the KRT.
+It has general description of the Version, a GRPC entrypoint for the Runtime Version and
+nodes that are connected between each other to form workflows that will be access through GRPC services defined on the entrypoint.  
+
+## Fields Specs
+
+Here is a description of each field divided in five main concepts that make a KRT file:
+
+### Metadata
+
+Descriptive information of the Runtime Version.
+
+- **version**: the unique ID of this version.
+- **krtVersion**: indicates the krt version this file is using. Only values allowed are "v1" and "v2".
+- **description**: brief text describing the content or functionality of this version.
+
+### Configuration
+
+This section describes all variables and files that are environment related and would be configured
+once the Runtime Version is uploaded to a KAI Server instance. This includes sensible data that should not
+be included on the KRT file, for example passwords.
+
+All configuration defined here is mandatory, the Runtime Version won't run if any of them is undefined.
+
+- **config**:
+  - **variables**: list of variable names that are going to be defined in KAI Server as environment variables.
+  - **files**: list of file names.
+
+### Entrypoint
+
+This section describes how to access the Runtime Version.
+
+- **entrypoint**:
+  - **proto**: protobuffer file describing messages and services contained in this Runtime Version.
+  - **image**: base image and tag used to run the entrypoint service. It's provided by Konstellation registry.
```

Comment:
Can you add a URL link to Konstellation registry?

Reply:
good idea!
added